apps/cloudflared/2024.11.1/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   cloudflared:
-    image: cloudflare/cloudflared:2024.11.1
+    image: cloudflare/cloudflared:2024.12.1
     container_name: ${CONTAINER_NAME}
     restart: always
     network_mode: 1panel-network
```

Comment:
There are no significant irregularities or potential issues in the provided code change. However, there is an unnecessary semicolon at the end of the `network_mode` line which could cause a syntax error if not removed.

Optimization suggestion: Remove the trailing semicolon from the `network_mode` line to enhance readability and correctness.